### PR TITLE
fix: do not use synthetic imports for nodejs core

### LINKED ISSUE #381: Error on TypeScript build: standard modules has no default export
Hi,

I'm currently building a TypeScript project using `googleapis` library (version `32.0.0`, with `google-auth-library` version `1.6.0`). Whenever I try to build my project, here are the errors I get:

```bash
node_modules/google-auth-library/build/src/auth/googleauth.d.ts(18,8): error TS1192: Module '"fs"' has no default export.
node_modules/google-auth-library/build/src/auth/googleauth.d.ts(19,8): error TS1192: Module '"http"' has no default export.
node_modules/google-auth-library/build/src/auth/googleauth.d.ts(20,8): error TS1192: Module '"stream"' has no default export.
node_modules/google-auth-library/build/src/auth/jwtaccess.d.ts(17,8): error TS1192: Module '"stream"' has no default export.
node_modules/google-auth-library/build/src/auth/jwtclient.d.ts(18,8): error TS1192: Module '"stream"' has no default export.
node_modules/google-auth-library/build/src/auth/oauth2client.d.ts(18,8): error TS1192: Module '"http"' has no default export.
node_modules/google-auth-library/build/src/auth/refreshclient.d.ts(17,8): error TS1192: Module '"stream"' has no default export.
```

It seems that TypeScript expects the `import * as http from 'http'` syntax for the standard modules.

I tried with both TypeScript **2.8** and **2.9**, same issue. Node version: **8.11**.

Could you look into this? Is there something wrong with my build configuration?

Best regards,

François Voron

### PR DESCRIPTION
**Pull request by JustinBeckwith**
Resolves #381

### CODE REVIEW

File: test/fixtures/kitchen/tsconfig.json [hunk 1]
```diff
@@ -2,13 +2,11 @@
   "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
     "rootDir": ".",
-    "outDir": "build"
+    "outDir": "build",
+    "esModuleInterop": false // test default setting
```
Comment (jinwoo) [inline test/fixtures/kitchen/tsconfig.json]: Why is this needed for tests?

Comment (JustinBeckwith) [reply test/fixtures/kitchen/tsconfig.json]: This ensures I won't break it again :) This is run as part of the kitchen sink.  If I used `import http from 'http'` now, this test would fail.
